<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-4IOPE15" failures="1" tests="1" name="Default test(failed)" time="0.114" errors="0" timestamp="2023-07-10T14:07:52 IST">
  <testcase classname="TestNg.Example7_SoftAssertClass1" name="TC5" time="0.025">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [hello] but found [hi],
	Failed2: act result is false-  expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [hello] but found [hi],
Failed2: act result is false-  expected [true] but found [false]
at TestNg.Example7_SoftAssertClass1.TC5(Example7_SoftAssertClass1.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TC5 -->
</testsuite> <!-- Default test(failed) -->
